src/TreeTransform.jl

```diff
@@ -177,7 +179,6 @@ function bottom_up_rewrite(
         # Transform each node until it reaches a fixed-point.
         any_changes = false
         for node in reverse(nodes)
```

Comment:
```suggestion
        for node in Iterators.reverse(nodes)
```
to avoid a copy. Will need an import up top to work

Reply:
Added `using Iterators: reverse`